centreon/packages/ui/src/components/CrudPage/Actions/Actions.styles.ts

```diff
@@ -2,7 +2,20 @@ import { makeStyles } from 'tss-react/mui';
 
 export const useActionsStyles = makeStyles()((theme) => ({
   search: {
-    maxWidth: theme.spacing(50)
+    maxWidth: theme.spacing(60),
```

Comment:
it might be worth to set a min-width like 180px or so in order to prevent a small search input